### DISCUSSION TITLE
Frame Scrollable in Two Directions

### DISCUSSION THREAD

**Discussion by DanielJOHara** [TomSchimansky/CustomTkinter#2129] [Ideas]
Are there plans to make a frame that can be scrolled both horizontally and vertically?
It would also be useful if other frames could be synchronised with the main frame. What I would ideally like is the ability to mimic the Excel freeze pane functionality. So frame at the top of and to the side of the two way scrollable frame that moved with the corresponding scroll bar.

**Comment by DanielJOHara** [comment]
I have developed a solution which I posted on a different thread. https://github.com/TomSchimansky/CustomTkinter/issues/2124#issuecomment-1854781819

**Comment by Akascape** [comment]
@DanielJOHara We also have CTkXYFrame https://github.com/Akascape/CTkXYFrame

**Reply by DanielJOHara** [reply]
Thanks for this, I needed the header and side titles to remain in view hence three frames with coordinated scrolling. I do need t add middle wheel scrolling so may be raiding code for ideas. 
Thanks for sharing
